AUTHORS

```diff
@@ -50,14 +64,18 @@ Martin Zachrison
 matija.mazi@gmail.com
 Matt Bogosian
 Matt Corallo
+Matthew Shaylor
 Michael Bell
 Michael Bumann
 Mike Hearn
 Mike Rosseel
 Miron Cuperman
+Miron Cuperman (devrandom)
 monk
 Mora Zyx
 mruddy
+Nicola
+Nicola Atzei
```

Comment:
No need to add the missing authors now, we do that before the next release. It's fine if you add yourself if you wish so.